packages/cashier/src/Containers/withdrawal.jsx

```diff
@@ -57,6 +60,11 @@ const Withdrawal = ({
         };
     }, [container, setActiveTab, setErrorMessage]);
 
+    React.useEffect(() => {
+        check10kLimit();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
```suggestion
    }, [check10kLimit]);
```
Here, we can avoid eslint-disable by adding `check10kLimit` to the dependency array. Reference to `check10kLimit` won't change since it is from the cashier store instance.